src/lplib/option.ml

```diff
@@ -8,15 +8,14 @@ let is_None : 'a t -> bool = fun o ->
 let get : 'a -> 'a option -> 'a = fun d o ->
   match o with None -> d | Some e -> e
 
+let bind : 'a t -> ('a -> 'b t) -> 'b t = Stdlib.Option.bind
+
```

Comment:
Can't we remove it completely if it is in stdlib? (Perhaps only after some ocaml version we have in CI?)